testing/ppsspp/APKBUILD

```diff
@@ -0,0 +1,81 @@
+# Contributor: Díaz Urbaneja Diego <sodomon2@gmail.com>
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=ppsspp
+pkgver=1.8.0
+pkgrel=0
+pkgdesc="PPSSPP - a fast and portable PSP emulator"
+url="https://www.ppsspp.org/"
+arch="x86_64 !x86" # only available x86_64 for compiling problems
+license="GPL-2.0-only"
+depends="cmake make"
```

Comment:
is cmake and make really required during runtime ?

Reply:
cmake don't believe it but make if required at runtime